hass_nabucasa/iot.py

```diff
@@ -135,6 +135,16 @@ async def async_handle_cloud(cloud, payload):
         _LOGGER.error(
             "You have been logged out from Home Assistant cloud: %s", payload["reason"]
         )
+    elif action == "disconnect_remote":
+        # Disconect Remote connection
+        await cloud.remote.disconnect(clear_snitun_token=True)
+    elif action == "user_notification":
```

Comment:
Can we include a `critcial_user_notification` too ? For now it would be the same logic but in the future it could include a push notification to the owner of the instance.
```suggestion
    elif action in ("user_notification", "critical_user_notification"):
```